integration-tests/tests/fixtures/integration_test_packages.py

```diff
@@ -0,0 +1,84 @@
+"""Define test packages fixtures."""
+
+import logging
+import subprocess
+from collections.abc import Iterator
+from typing import Literal
+
+import pytest
+
+from tests.utils.config import (
+    PackageConfig,
+    PackageRun,
+)
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture(scope="session")
```

Comment:
Probably shouldn't be a session scoped fixture now that I look at it. We may need to startup and teardown multiple CLP packages during a single pytest invocation, as we may need to test different clp configs.